src/main/java/frc/swerve/Swerve.java

```diff
@@ -17,6 +17,7 @@
 	private final double RADIUS;
 	private final double WHEEL_ANGLE;
 	private final double ROTATE_SCALE;
+	/** proportional scaling constant */
```

Comment:
Please add a similar comment to L22 for the "derivative gain" or "derivative scaling constant." If you're confused about k_P and k_D, come find me one of these Fridays, and we can get into control systems.